x/ccv/provider/keeper/misbehaviour.go

```diff
@@ -142,8 +142,20 @@ func headerToLightBlock(h ibctmtypes.Header) (*tmtypes.LightBlock, error) {
 }
 
 // CheckMisbehaviour checks that headers in the given misbehaviour forms
-// a valid light client attack and that the corresponding light client isn't expired
+// a valid light client attack from an ICS consumer chain and that the light client isn't expired
 func (k Keeper) CheckMisbehaviour(ctx sdk.Context, misbehaviour ibctmtypes.Misbehaviour) error {
+	// check that the misbehaviour is for an ICS consumer chain
+	clientID, found := k.GetConsumerClientId(ctx, misbehaviour.Header1.Header.ChainID)
```

Comment:
nit: Maybe rename `clientID` to `clientId` to be consistent with the `GetConsumerClientId`.

Reply:
Sure changed in last commit